deployment/terraform/task-definitions/app.json

```diff
@@ -2,8 +2,18 @@
   {
     "name": "django",
     "image": "${image}",
-    "cpu": ${cpu},
-    "memory": ${memory},
```

Comment:
Specifying these limitations within the task definition template is redundant with Fargate.